2-projects/1-todo-list/script.js

```diff
@@ -13,13 +43,33 @@ let todos = [
 populateTodoList(todos);
 
 // This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
+let addBtn = document.querySelector(".btn");
+addBtn.addEventListener("click", addNewTodo);
+
 function addNewTodo(event) {
-  // The code below prevents the page from refreshing when we click the 'Add Todo' button.
-  event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+  event.preventDefault();
+  let input = document.getElementById("todoInput");
+
+  //todos.push({ task: input.value, completed: false });
+  populateTodoList([{ task: input.value, completed: false }]);
```

Comment:
You could pass the whole todos lists, like you have in the commented line above. It just means that in populateTodoList you need to replace the contents of the `ul`, rather than appending new items to what is already there.